Architecture_Engine/Architecture_Engine.csproj

```diff
@@ -89,6 +89,7 @@
     <Compile Include="Create\Theatron\TierProfile.cs" />
     <Compile Include="Create\Theatron\VomitoryParameters.cs" />
     <Compile Include="Modify\CleanRoom.cs" />
+    <Compile Include="Modify\RoundingVertices.cs" />
```

Comment:
Recompile the code after deleting the file to get the new csproject file and repush that to fix the CI checks please :smile: